### PR TITLE
24773 deprecation of getrequestformat

### PR DESCRIPTION
**Pull request by mattwillo**
| Q             | A
| ------------- | ---
| Branch?       | 4.1
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | yes
| Tests pass?   | yes
| Fixed tickets | #24773 
| License       | MIT
| Doc PR        | symfony/symfony-docs#...

### CODE REVIEW

**Comment by stof** [comment]
Given that stable release of Symfony 3.4 and 4.0 is tomorrow, this is far too late to add a new deprecation in it. So I changed the target to master

**Comment by stof** [comment]
referencing your own PR in `fixed ticket` does not make sense

File: src/Symfony/Component/HttpFoundation/Request.php [hunk 1]
```diff
@@ -196,7 +196,7 @@ class Request
     /**
      * @var string
      */
-    protected $format;
+    protected $requestedResponseFormat;
```
Comment (stof) [inline src/Symfony/Component/HttpFoundation/Request.php]: Renaming a protected property is a BC break. This is not allowed. See https://symfony.com/doc/current/contributing/code/bc.html for details about which kind of changes are allowed or no to respect the BC policy.

**Comment by Jean85** [comment]
Status: needs work

File: src/Symfony/Component/HttpFoundation/Request.php (hunk 1 above)
Comment (mattwillo) [reply src/Symfony/Component/HttpFoundation/Request.php]: Will revert that.

**Comment by mattwillo** [comment]
Updated above link, it was meant to point to the issue not my own PR, that was very strange.

Reverted rename of protected $format.

**Comment by mattwillo** [comment]
Status: Needs review

**Comment by mattwillo** [comment]
I am investigating the break on 7.2

**Comment by nicolas-grekas** [comment]
I'm closing here as this didn't get much traction: renaming methods needs a clear benefit, so that the extra work we put on devs is justified. it doesn't feel the case here.
Thanks anyway for your PR, waiting the next one :)

**Pull request closed without merging**